recipes/rmm/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+import os
+import textwrap
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import export_conandata_patches, get, copy, replace_in_file, save
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.52.0"
+
+
+class RmmConan(ConanFile):
+    name = "rmm"
+    description = "RAPIDS Memory Manager"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/rapidsai/rmm"
+    topics = ("cuda", "memory-management", "memory-allocation", "rapids", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
```suggestion
    settings = "os", "arch", "compiler", "build_type"
    no_copy_source = True
```
Avoid copying source files to the build folder, as you are not building something, only copying to the package folder . 

https://docs.conan.io/2/reference/conanfile/attributes.html#no-copy-source